docs/blog/2019-03-11-dot-org-prototypes/index.md

```diff
@@ -0,0 +1,139 @@
+
+### Roles:
+
+We gathered a couple engineers, a product manager, and a product designer and assigned a couple special roles:
+
+1.  Timer: someone to use a timer to make sure people move through the activities below.
+2.  Note taker: at the end, someone needs to take notes during the “final feedback” section.
+
+### Review design requirements
+
+We briefly reminded ourselves of what the current homepage looks like on mobile, and then review the following information on the requirements for redesigning the homepage:
+
+- Optimize for the purpose of the homepage: to help as many developers as possible use Gatsby
+- How do we know if we’re succeeding in that purpose? Increase in number of clicks on Get Started button and monthly active users growing
+- What should the homepage communicate about Gatsby? “Gatsby is a free, open-source, modern website framework based on React to create and deploy blazing fast websites or apps with ease.”
+- Who is the audience? developers who build sites for clients, companies, and their own projects
+- What are our value propositions for that audience? fast sites, performance out of the box, productive from the start
+
+### Sketching
+
+We did 3 rounds of the following pattern and saw that in round two, our sketches diverged and in round three, they converged:
+
+- 4 minutes: Everyone draw (with pen/pencil & paper) what they think the mobile version of the .org homepage ought to look like.
+- 1.5 minutes: Upload picture to this Slack channel: #p-dot-org-prototypes
+- 1 minute each: The person who drew the sketch being talked about can explain it for 1 minute, focusing only on facts. No pitching your ideas or persuading people.
+
+### Final feedback after sketching rounds (need a note taker to record this feedback):
+
+- 2 minutes per person in which they share comments in these categories:
+- Factual observations about this exercise.
+- What I liked about these sketches and why.
+- What I don’t like about these sketches and why.
+- What I don’t understand.
+
+## What did we conclude from the prototype workshop?
+
+We coalesced around:
+The homepage should be ordered by explaining the following:
+
+- What is Gatsby?
+- Why is it worth paying attention to?
+- How do you use it?
+
+We left the prototype workshop with a few concerns that could be validated through testing:
+
+- Logos of companies could turn off developers b/c it’s cliche and sometimes meaningless or they could provide social proof that world-class engineers use Gatsby. We’ll test this. Possibility is to combine the logos with case studies so they aren’t in the classic “logos floating on white space” design that most startups use.
+- Debate over how to help developers know that Gatsby also has some upcoming commercial offerings. What do we call the commercial offerings so they make sense to first time visitors?
+
+### Settled concerns:
+
+"Install Gatsby" is an more specific call to action, so better than “Get Started” probably.
```

Comment:
Better than "Get Started" how?  It seems like the inverse is true for .org homepage messaging...?